packages/api/lib/xunit-parser.js

```diff
@@ -93,21 +89,7 @@ class Parser {
   * @returns {string} the string following `:repo`, which may be empty
   */
   trim (url) {
-    const updated = url.replace(/(.)*github.com\/[a-zA-Z-]*\/[a-zA-Z-]*/g, '');
-    if (updated.length > 1) return updated.substring(1); // Get rid of starting `/`
-    return updated;
-  }
-
-  // IMPORTANT: All values that will be used as keys in Firestore must be escaped with the firestoreEncode function
-  /**
-  * Parse the build data and convert it into a JSON format that can easily be read and stored.
-  * @param metadata Contains all data sent to the endpoint besides the actual test results.
-  * @returns {JSON} The metadata for the build
-  */
-  cleanXunitBuildInfo (metadata) {
-    metadata.repoId = firebaseEncode(metadata.repoId);
-    metadata.buildId = firebaseEncode(metadata.buildId);
-    return metadata;
+    return url.replace(/^github.com\/[^/]+\/[^/]+\/?/, '');
```

Comment:
This is so much better than my mangled attempt at regex!

Reply:
@cedpeters practice makes perfect 😄 regex is a great thing to keep getting better at.